docs/docker/DHCP.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Docker DHCP and Network Modes
+description: Setting up DHCP for Docker Pi-hole
+---
```

Comment:
Can you add `last_updated` to the frontmatter? Just a date is fine, no need for the time.